tox.ini

```diff
@@ -73,7 +73,7 @@ deps =
 depends =
     py36: clean
 commands =
-    pytest -v --maxfail=2 --cov-config=.coveragerc --cov=bookstore -W always bookstore/tests/
+    pytest -v --maxfail=2 --cov-config=.coveragerc --cov=bookstore -W always
```

Comment:
I would be inclined to leave the directory here and move the client tests to `bookstore/tests/client/...`